CONTRIBUTING.md

```diff
@@ -107,15 +128,17 @@ Contributing code
 
 We gladly accept contributions via GitHub pull requests.
 
+Please peruse our
+[style guides](https://flutter.io/style-guide/) and
```

Comment:
Is this still the correct URL For the style guide?  I thought we moved it to the wiki.